packages/grafana-ui/src/components/DateTimePickers/RelativeTimeRangePicker/RelativeTimeRangePicker.tsx

```diff
@@ -123,7 +122,7 @@ export function RelativeTimeRangePicker(props: RelativeTimeRangePickerProps) {
         </span>
       </button>
       {isOpen && (
-        <Portal>
```

Comment:
I had to remove the usage of `Portal` which was introduced in this [PR](https://github.com/grafana/grafana/pull/60016) because with it the time options were unable to be clicked as the modal would be closed immediately on any click. Example: 
![2023-06-30 17 53 00](https://github.com/grafana/grafana/assets/6271380/8bcc27b3-2987-4b25-99ca-55d83353ec30)

It seems like the original bug that was the reason it was added in the first place is not happening anymore with this new redesign so it looked safe to remove it.